### PR TITLE
Added advanced material to the book

### PR DESCRIPTION
**Pull request by Mirabellensaft**

### CODE REVIEW

File: embedded-workshop-book/src/SUMMARY.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Summary
+
+- [Intro](./intro.md)
```
Comment (Lotterleben) [inline embedded-workshop-book/src/SUMMARY.md:3]: nitpick: the almost empty page looks very Work In Progress-y, I'd rather comment it out until we have content for it

File: embedded-workshop-book/src/hardware.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ In this workshop we'll use both the nRF52840 Development Kit (DK) and the nRF528
 
 For the span of this workshop keep the nRF52840 DK connected to your PC using a micro-USB cable. Connect the USB cable to the J2 port on the nRF52840 DK. Instructions to identify the USB ports on the nRF52840 board can be found in the top level README file.
 
+[❗️link or crosspost J2 instructions, for less clicking]
```
Comment (Lotterleben) [inline embedded-workshop-book/src/hardware.md:7]: question: are these todos?

File: embedded-workshop-book/src/intro.md [hunk 3]
```diff
@@ -4,3 +4,9 @@
 
 ### in development
 ❗️ To do
+
```
Comment (Lotterleben) [inline embedded-workshop-book/src/intro.md:7]: could we link the setup instructions (i.e. https://github.com/ferrous-systems/embedded-trainings-2020/blob/master/README.md) either here or at the beginning of the Beginner/Advanced Course chapters?

**Comment by Lotterleben** [comment]
above comments are non-blocking, fwiw

File: embedded-workshop-book/src/hardware.md (hunk 2 above)
Comment (Mirabellensaft) [reply embedded-workshop-book/src/hardware.md:7]: To dos and notes, that I am currently working on and removing

**Pull request merged**